README.md

```diff
@@ -2,12 +2,12 @@
 
 This repository provides a sample ROS2 environment working on a Jetson-based ECU and GMSL2-compatible cameras.
 
-As a sample application, the following images show object recognition results using the contents of this repository. Various perception applications will be added in
+As a sample application, the following images show object detection results using the contents of this repository. Various perception applications will be added in
 the future and you can develop them in this provided environment.
 
-[Edge.Auto](http://edge.auto) sensor fusion system can be realized by using this with [edge-auto](https://github.com/tier4/edge-auto) repository.
+[Edge.Auto](https://sensor.tier4.jp/) sensor fusion system can be realized by using this with [edge-auto](https://github.com/tier4/edge-auto) repository.
 
-![object recognition example](docs/sample.png "edge-auto-jetson object recognition example")
+![object detection example](docs/sample.png "edge-auto-jetson object detection example")
 
 # Prerequisites
```

Comment:
Could you add one more `#` (i.e., change this line's level to h2) because pre-commit.ci points out this might be error.